### PR TITLE
Added Unique Field Attribute Doc

### PR DESCRIPTION
**Pull request by EpicKris**
Issue: #2989

Added unique field attribute to the documentation.

### CODE REVIEW

File: user_guide_src/source/database/forge.rst [hunk 1]
```diff
@@ -97,6 +97,7 @@ Additionally, the following key/values can be used:
 -  auto_increment/true : generates an auto_increment flag on the
    field. Note that the field type must be a type that supports this,
    such as integer.
+-  unique/false : to generate a unique key for the field definition.
```
Comment (narfbg) [inline user_guide_src/source/database/forge.rst]: Why does this say 'false'?

**Comment by narfbg** [comment]
Also, not that the branch name you use matters, but this isn't a "hotfix".

File: user_guide_src/source/database/forge.rst [hunk 2]
```diff
@@ -97,6 +97,7 @@ Additionally, the following key/values can be used:
 -  auto_increment/true : generates an auto_increment flag on the
    field. Note that the field type must be a type that supports this,
    such as integer.
+-  unique/true : to generate a unique key for the field definition.
```
Comment (EpicKris) [inline user_guide_src/source/database/forge.rst]: Changed to true.

**Comment by EpicKris** [comment]
@narfbg What branch name should I have used? `feature`?

**Comment by narfbg** [comment]
As I said, the branch name itself is irrelevant - you could name it "pepsi" if you want to. :)

Just saying that a hotfix is something completely different to what you did here.

**Pull request merged**